### PR TITLE
[translations] Update French from an alternative source

### PR DESCRIPTION
**Pull request by rtsisyk**
Signed-off-by: Roman Tsisyk <roman@tsisyk.com>

----------

I've got this French translation from a company that wanted to translate our Google Play listings. I am not an expert in French. @organicmaps/translations-fr, could you please review it and tell me if it's better or worse?

### CODE REVIEW

File: android/src/fdroid/play/listings/fr-FR/full-description.txt [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-Organic Maps est <b>pur et biologique, fait avec amour</b> :
+Organic Maps est <b>pur et biologique, fait avec amour</b>:
 
-• Une expérience hors ligne ultra-rapide
+• Expérience hors ligne ultra-rapide
 • Respecte votre vie privée
 • Économise votre batterie
 • Pas de frais de données mobiles inattendus
-• Simplicité d'utilisation, avec uniquement les fonctions les plus importantes.
+• Simple à utiliser, avec uniquement les fonctions les plus importantes incluses
```
Comment (Jean-BaptisteC) [inline android/src/fdroid/play/listings/fr-FR/full-description.txt:26]:
```suggestion
• Simple d'utilisation, avec uniquement les fonctions les plus importantes incluses
```

File: android/src/fdroid/play/listings/fr-FR/full-description.txt [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
 
-Sans trackers et autres mauvaises choses :
+Libre de suivis et autres mauvaises choses :
```
Comment (Jean-BaptisteC) [inline android/src/fdroid/play/listings/fr-FR/full-description.txt:28]:
```suggestion
Sans pisteurs et autres mauvaises choses :
```

File: android/src/fdroid/play/listings/fr-FR/full-description.txt [hunk 3]
```diff
@@ -1,52 +1,52 @@
...
-• Pas de pistage
+• Pas de suivi
 • Pas de collecte de données
-• Pas d'appel au domicile
-• Pas d'inscription fastidieuse
+• Pas de téléphone à la maison
+• Pas d'inscription ennuyeuse
 • Pas de tutoriels obligatoires
-• Pas de spam excessif
+• Pas de spam par courriel bruyant
```
Comment (Jean-BaptisteC) [inline android/src/fdroid/play/listings/fr-FR/full-description.txt:36]:
```suggestion
• Pas de spam abusif
```

File: android/src/fdroid/play/listings/fr-FR/full-description.txt [hunk 4]
```diff
@@ -1,52 +1,52 @@
...
 • Pas de notifications push
-• Pas de logiciels malveillants
-• N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Entièrement bio
+• Pas de logiciel promotionnel
+• N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Purement biologique
 
-Chez Organic Maps, nous pensons que la <b>vie privée est un droit humain fondamental</b> :
+Chez Cartes biologiques, nous pensons que <b>la confidentialité est un droit humain fondamental</b>:
```
Comment (Jean-BaptisteC) [inline android/src/fdroid/play/listings/fr-FR/full-description.txt:41]:
```suggestion
Chez Organic Maps, nous pensons que <b>la vie privée est un droit humain fondamental</b>:
```

File: android/src/fdroid/play/listings/fr-FR/full-description.txt [hunk 5]
```diff
@@ -1,52 +1,52 @@
...
 
-• Organic Maps est un projet indépendant à code source ouvert dirigé par la communauté.
-• Nous protégeons la vie privée des regards indiscrets des géants de la tech
-• Restez en sécurité, où que vous soyez
+• Cartes biologiques est un projet source-ouverte piloté par la communauté indépendante
```
Comment (Jean-BaptisteC) [inline android/src/fdroid/play/listings/fr-FR/full-description.txt:43]:
```suggestion
• Organic Maps est un projet open-source géré par une communauté indépendante
```

File: android/src/fdroid/play/listings/fr-FR/full-description.txt [hunk 6]
```diff
@@ -1,52 +1,52 @@
...
+• Nous protégeons la vie privée des regards indiscrets de Big Tech
+• Restez en sécurité où que vous soyez
 
-Selon Exodus Privacy Report, il n'y a aucun traceur et les autorisations requises sont minimales.
+Zéro localisateurs de suivi et seules les autorisations minimales requises sont trouvées selon Exodus Privacy Report.
```
Comment (Jean-BaptisteC) [inline android/src/fdroid/play/listings/fr-FR/full-description.txt:47]:
```suggestion
Zéro pisteurs et seules les autorisations minimales requises sont trouvées dans le rapport d'Exodus Privacy.
```

**Review by Jean-BaptisteC — CHANGES_REQUESTED** [review]
Some changes :)

**Comment by biodranik** [comment]
@Jean-BaptisteC thanks! @rtsisyk I told you that these Google Play translators do not worth their money.

**Comment by Jean-BaptisteC** [comment]
I think this company have use Google translate to translate this description :(
Because Organic Maps is translated by `Cartes biologiques` and trackers by `localisateurs de suivi` 😂
My main language is french but i think a second review is necessary :)

**Comment by rtsisyk** [comment]
> I think this company have use Google translate to translate this description :( Because Organic Maps is translated by `Cartes biologiques` and trackers by `localisateurs de suivi` 😂 My main language is french but i think a second review is necessary :)

This is what I thought, but wanted to validate. So, your conclusion that this new translation is substandard?

**Comment by arnaudvergnet** [comment]
Can confirm those translations are pretty shit

File: android/src/fdroid/play/listings/fr-FR/full-description.txt [hunk 7]
```diff
@@ -1,52 +1,52 @@
...
+• Pas de téléphone à la maison
+• Pas d'inscription ennuyeuse
 • Pas de tutoriels obligatoires
-• Pas de spam excessif
+• Pas de spam par courriel bruyant
 • Pas de notifications push
-• Pas de logiciels malveillants
-• N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Entièrement bio
+• Pas de logiciel promotionnel
+• N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Purement biologique
```
Comment (arnaudvergnet) [inline android/src/fdroid/play/listings/fr-FR/full-description.txt:39]:
```suggestion
• S̶a̶n̶s̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Purement biologique
```

File: android/src/fdroid/play/listings/fr-FR/full-description.txt [hunk 8]
```diff
@@ -1,52 +1,52 @@
...
 
-Chez Organic Maps, nous pensons que la <b>vie privée est un droit humain fondamental</b> :
+Chez Cartes biologiques, nous pensons que <b>la confidentialité est un droit humain fondamental</b>:
 
-• Organic Maps est un projet indépendant à code source ouvert dirigé par la communauté.
-• Nous protégeons la vie privée des regards indiscrets des géants de la tech
-• Restez en sécurité, où que vous soyez
+• Cartes biologiques est un projet source-ouverte piloté par la communauté indépendante
+• Nous protégeons la vie privée des regards indiscrets de Big Tech
```
Comment (arnaudvergnet) [inline android/src/fdroid/play/listings/fr-FR/full-description.txt:44]:
```suggestion
• Nous protégeons la vie privée des regards indiscrets des géants de la tech
```

File: android/src/fdroid/play/listings/fr-FR/full-description.txt (hunk 6 above)
Comment (arnaudvergnet) [reply android/src/fdroid/play/listings/fr-FR/full-description.txt:47]:
```suggestion
Zéro trackers et seules les autorisations minimales requises sont trouvées dans le rapport d'Exodus Privacy.```

**Review by arnaudvergnet — CHANGES_REQUESTED** [review]

**Comment by arnaudvergnet** [comment]
The old translation was often better than this one, even google translate does a better job.

**Comment by rtsisyk** [comment]
OK, I am closing this PR then. Thanks for you feedback!

**Pull request closed without merging**